README.md

```
@@ -4,36 +4,37 @@
 > [!Important]
 >
-> This page is meant for the *development* of Slipshow. If you are a user, head to the official website: https://slipshow.org
+> This page is about the *development* of Slipshow; if you are looking to *use* it, head to the official website at [slipshow.org](https://slipshow.org).
 
-## **Slipshow and scrolling presentations.**
+## Slipshow is all about *scrolling* presentations
 
-In a slipshow presentation, the equivalent of a slide is called a _slip_. Each slip is like a slide, but with no bottom limit. That is, the content can be arbitrarily long! During the presentation, the camera will "scroll" down to reveal the hidden content, following a script given by the presenter!
+In a Slipshow presentation, the equivalent of a slide is called a _slip_. Each slip is like a slide, but with no bottom limit. That is, the content can be arbitrarily long! During the presentation, the camera will "scroll" down to reveal the hidden content, following a script given by the presenter!
 
 ![](slip_scroll.gif)
```

markdownlint-cli2 (0.23.0)

[warning] Images should have alternate text (alt text)

(MD045, no-alt-text)